### PR TITLE
[zh-cn] sync install-kubeadm.md

### PR DESCRIPTION
**Pull request by asa3311**
sync install-kubeadm.md

### CODE REVIEW

**Comment by tengqm** [comment]
Please check our style guide on Chinese localizations [here](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

**Comment by asa3311** [comment]
> Please check our style guide on Chinese localizations [here](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

已调整缩进。

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * 节点之中不可以有重复的主机名、MAC 地址或 product_uuid。请参见[这里](#verify-mac-address)了解更多详细信息。
 * 开启机器上的某些端口。请参见[这里](#check-required-ports)了解更多详细信息。
 * 禁用交换分区。为了保证 kubelet 正常工作，你**必须**禁用交换分区。
+  * 例如，`sudo swapoff -a` 将暂时禁用交换分区。要使此更改在重启后保持不变，请确保在如 `/etc/fstab`、`systemd.swap` 等配置文件中禁用交换分区，具体取决于你的系统如何配置。
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:54]:
```suggestion
  * 例如，`sudo swapoff -a` 将暂时禁用交换分区。要使此更改在重启后保持不变，请确保在如 
    `/etc/fstab`、`systemd.swap` 等配置文件中禁用交换分区，具体取决于你的系统如何配置。
```

**Pull request merged**